scripts/common.py

```diff
@@ -222,6 +222,18 @@
     'translation-admin-tests/remove-translation-rights'
 ]
 
+MOBILE_ACCEPTANCE_TESTS_SUITE_NAMES = [
```

Comment:
See comment in the actions, can we unify with non-mobile tests so that we always treat desktop and mobile on the same footing?

Reply:
Done